### DISCUSSION TITLE
Docs:  Added a command for magic support when using fish

### DISCUSSION THREAD

**Discussion by 0123takaokeita** [pkgxdev/pkgx#661] [Ideas]
I found that some commands failed when I executed them as described in the readme.

I found out that the writing style is different for fish.

I ran into this because I was using fish, but how about adding this for others?

fail
<img width="415" alt="CleanShot 2023-07-29 at 23 21 05@2x" src="https://github.com/teaxyz/cli/assets/65272471/db8c142d-6333-4c77-9d4c-5b8b21ae5462">

success
<img width="259" alt="CleanShot 2023-07-29 at 23 20 45@2x" src="https://github.com/teaxyz/cli/assets/65272471/1bbf350f-dfb4-4ed7-bd96-ff748ea01df3">

**Comment by mxcl** [comment]
With v1 we need shellcode for fish (fresh). We also typically recommend `eval "$(tea foo)"` now which is POSIX portable.
